### PR TITLE
Move reviews from core into local

### PR DESCRIPTION
**Pull request by ryanmitchell**
WIP - see PR in core for progress

### CODE REVIEW

File: Extension.php [hunk 1]
```diff
@@ -15,6 +17,19 @@ public function register()
...
+            'reviews' => 'Igniter\Local\Models\Reviews_model',
+        ]);
+        
+        Orders_model::extend(function ($model) {
+            $model->relation['morphMany']['review'] = ['Igniter\Local\Models\Reviews_model'];
+        });
+        
+        Locations_model::extend(function ($model) {
+            $model->relation['hasMany']['reviews'] = ['Igniter\Local\Models\Reviews_model'];
+            $model->allowedSortingColumns = array_merge($model->allowedSortingColumns, ['reviews_count asc', 'reviews_count desc',]);
```
Comment (sampoyigi) [inline Extension.php:31]: Was wondering why you made the property public.. 😎 👍

File: Extension.php [hunk 2]
```diff
@@ -98,4 +113,29 @@ public function registerImportExport()
...
+                    'permission' => 'Admin.Reviews',
+                ],
+            ]
+        ];
+    }
+
+    public function registerPermissions()
+    {
+        return [
+            'Igniter.Local.Reviews' => [
```
Comment (sampoyigi) [inline Extension.php:135]: This should be `Admin.Reviews`

**Review by sampoyigi — CHANGES_REQUESTED** [review]

File: Extension.php (hunk 1 above)
Comment (ryanmitchell) [reply Extension.php]: Yeah it wasn't enough - I've had to make some other changes which you may not like 😁 
Let me know if there is a better way.

File: models/LocalSettings.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Igniter\Local\Models;
+
+use Model;
+
+class LocalSettings extends Model
```
Comment (sampoyigi) [inline models/LocalSettings.php:7]: Maybe ReviewSettings to be more specific?

File: Extension.php (hunk 1 above)
Comment (sampoyigi) [reply Extension.php]: Yep, you can leave it as static and merge the array outside the extend closure/callback.

**Review by sampoyigi — CHANGES_REQUESTED** [review]

**Comment by ryanmitchell** [comment]
Cheers - those are done.
That just leaves the outstanding questions from the core PR.

File: Extension.php [hunk 4]
```diff
@@ -15,6 +18,20 @@ public function register()
...
+
+        Orders_model::extend(function ($model) {
+            $model->relation['morphMany']['review'] = ['Igniter\Local\Models\Reviews_model'];
+        });
+
+        Locations_model::extend(function ($model) {
+            $model->relation['hasMany']['reviews'] = ['Igniter\Local\Models\Reviews_model'];
+        });
+
+        Locations_model::$allowedSortingColumns = array_merge(Locations_model::$allowedSortingColumns, ['reviews_count asc', 'reviews_count desc']);
```
Comment (sampoyigi) [inline Extension.php:34]: Put these within the boot method

**Review by sampoyigi — CHANGES_REQUESTED** [review]

**Comment by ryanmitchell** [comment]
Should be all those changes done

**Review by sampoyigi — APPROVED** [review]

**Pull request merged**